docs/en/guides/workouts-monitoring.md

```diff
@@ -40,91 +40,90 @@ Monitoring workouts through pose estimation with [Ultralytics YOLOv8](https://gi
 
         ```python
         from ultralytics import YOLO
-        from ultralytics.solutions import ai_gym
+        from ultralytics import solutions
         import cv2
-
+        
         model = YOLO("yolov8n-pose.pt")
         cap = cv2.VideoCapture("path/to/video/file.mp4")
         assert cap.isOpened(), "Error reading video file"
         w, h, fps = (int(cap.get(x)) for x in (cv2.CAP_PROP_FRAME_WIDTH, cv2.CAP_PROP_FRAME_HEIGHT, cv2.CAP_PROP_FPS))
-
-        gym_object = ai_gym.AIGym()  # init AI GYM module
-        gym_object.set_args(line_thickness=2,
-                            view_img=True,
-                            pose_type="pushup",
-                            kpts_to_check=[6, 8, 10])
-
+        
+        solutions.configure(names=model.names, line_thickness=3, pose_type="squat",
+                      view_img=True, kpts_to_check=[5, 11, 13])
+        
+        # init AI GYM module
```

Comment:
@RizwanMunawar we should always capitalize the start of single-line comments like this one